### PR TITLE
Adding deploy_remote_cc module in submodules

### PR DESCRIPTION
**Pull request by prashantgajare**
Hi @badjava @dickolsson ,

Good Morning,

As discussed with @badjava , raising the PR of submodule `deploy_remote_cc` which clears the drupal caches of the remote environment using deploy endpoints of deploy module authenticated by `DeployAuthenticator`. 

Please review the PR and let me know in case of any concerns.

Thanks!

### CODE REVIEW

File: modules/deploy_remote_cc/deploy_remote_cc.module [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  );
+}
+
+/**
+ * Implements hook_permission().
+ */
+function deploy_remote_cc_permission() {
+  return array(
+    'clear remote caches' => array(
+      'title' => t('Access of Remote Cache Clear'),
```
[the remark this answers is not in the document]
Comment (prashantgajare) [reply modules/deploy_remote_cc/deploy_remote_cc.module]: @timmillwood yes agreed.
"Clear remote caches" is crisp and clear, made the changes.

File: modules/deploy_remote_cc/includes/deploy_remote_cc.admin.inc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+/**
+ * Admin settings form.
+ */
+function deploy_remote_cc_endpoint_settings($form, $form_state) {
+
+  $endpoints = deploy_endpoint_load_all();
+
+  if (!count($endpoints)) {
+    $msg = t('No deploy endpoints configured.');
+    if (module_exists('deploy_ui') && user_access('administer deployment plans')) {
```
[the remark this answers is not in the document]
Comment (prashantgajare) [reply modules/deploy_remote_cc/includes/deploy_remote_cc.admin.inc]: @timmillwood Done with the changes. Please review.

**Comment by prashantgajare** [comment]
@suzymasri Done with the changes, Please review and validate.

Thanks!

**Comment by saurabhtcs** [comment]
@suzymasri Just want to check if  you got the time to validates the changes.  By when we expect this module in repo.

**Comment by timmillwood** [comment]
@prashantgajare I was just about to commit, but the Travis test is failing. It seems unrelated, but the 7.x-3.x branch is passing. Can you make the changes needed for Travis to pass?

**Comment by prashantgajare** [comment]
@timmillwood @vijaycs85 Issue is solved and travis check is passed now on the branch. The issue was due to `dist:trusty` used in 7.x-3.x branch which was changed earlier from `dist:trusty` to `dist:precise`, as `dist:precise` is going to get decommissioned. For now I have changed the `dist:precise` to find the root cause of the failing travis build.

Let me know your thoughts which `dist` needs to be used in this branch.

**Comment by vijaycs85** [comment]
@prashantgajare ah, that rings a bell on Travis blog sometime back. good catch 👍

**Pull request merged**